ClassPaulita/GuessTheCardGame.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+
+public class GuessTheCardGame
+{  
+    private bool IsValidInput(string input)
+    {
+        if (input == null || input.Length < 1)
+            return false;
+
+        char choice = input[0];
+
+        // if user chose to play.
+        if (choice == 'p' || choice == 'P')
```

Comment:
All of these if's are duplicate code (they are the same as used in the Run function. 

You should create a function for each of these if's

Like 
```
UserChosePlay(char choice)
{
     return choice == 'p' || choice == 'P';
}
```